tests/installation/bootloader_hyperv.pm

```diff
@@ -211,6 +211,8 @@ sub run {
 
     hyperv_cmd("$ps Set-VMProcessor $name -Count $cpucount");
 
+    # Set ram for the VM
+    hyperv_cmd("$ps Set-VMMemory $name -DynamicMemoryEnabled \$true -MinimumBytes 256MB -StartupBytes 512MB -MaximumBytes ${ramsize}MB");
```

Comment:
I wonder what's the default when this line is not called and where it is defined. We could of course just call this command when the value of requested RAM differs from default but I guess being explicit is better here.